### ISSUE TITLE
Torrentbytes fails during setup with no error message

### ISSUE THREAD

**Issue by adamwinn** [Jackett/Jackett#1056]
I tried to setup Torrentbytes and it failed after I entered my credentials with no error message.

http://imgur.com/a/Axwqf

Jackett version:  0.7.1006.0
Mono version (if not using Windows):  4.6.2

-------------------------------
```
02-12 19:27:25 Info WebAPIRequestLogger SendAsync  
System.Web.Http.MessageHandlers: WebAPIRequestLogger SendAsync 
02-12 19:27:25 Info   Content-type='application/json; charset=utf-8', content-length=1027 
System.Web.Http.Request:   Content-type='application/json; charset=utf-8', content-length=1027
02-12 19:27:25 Info AdminController Dispose  
System.Web.Http.Controllers: AdminController Dispose 
02-12 19:27:25 Info AdminController Dispose  
System.Web.Http.Controllers: AdminController Dispose 
02-12 19:27:37 Info   http://server:9117/Admin/configure_indexer 
System.Web.Http.Request:   http://server:9117/Admin/configure_indexer
02-12 19:27:37 Info WebAPIRequestLogger SendAsync  
System.Web.Http.MessageHandlers: WebAPIRequestLogger SendAsync 
{"indexer":"torrentbytes","name":"TorrentBytes","config":[{"id":"sitelink","value":"https://www.torrentbytes.net/"},{"id":"username","value":"username"},{"id":"password","value":"password"},{"id":""},{"id":"cookieheader","value":""},{"id":"lasterror","value":""}]}
02-12 19:27:37 Debug POST: http://server:9117/Admin/configure_indexer 
02-12 19:27:37 Debug Body: {"indexer":"torrentbytes","name":"TorrentBytes","config":[{"id":"sitelink","value":"https://www.torrentbytes.net/"},{"id":"username","value":"username"},{"id":"password","value":"password"},{"id":""},{"id":"cookieheader","value":""},{"id":"lasterror","value":""}]} 
02-12 19:27:37 Info DefaultHttpControllerSelector SelectController Route='MS_SubRoutes:System.Web.Http.Routing.IHttpRouteData[]' 
System.Web.Http.Controllers: DefaultHttpControllerSelector SelectController Route='MS_SubRoutes:System.Web.Http.Routing.IHttpRouteData[]'
02-12 19:27:37 Info DefaultHttpControllerSelector SelectController Admin 
System.Web.Http.Controllers: DefaultHttpControllerSelector SelectController Admin
02-12 19:27:37 Info HttpControllerDescriptor CreateController  
System.Web.Http.Controllers: HttpControllerDescriptor CreateController 
02-12 19:27:37 Info DefaultHttpControllerActivator Create  
System.Web.Http.Controllers: DefaultHttpControllerActivator Create 
02-12 19:27:37 Info DefaultHttpControllerActivator Create Jackett.Controllers.AdminController 
System.Web.Http.Controllers: DefaultHttpControllerActivator Create Jackett.Controllers.AdminController
02-12 19:27:37 Info HttpControllerDescriptor CreateController Jackett.Controllers.AdminController 
System.Web.Http.Controllers: HttpControllerDescriptor CreateController Jackett.Controllers.AdminController
02-12 19:27:37 Info AdminController ExecuteAsync  
System.Web.Http.Controllers: AdminController ExecuteAsync 
02-12 19:27:37 Info ApiControllerActionSelector SelectAction  
System.Web.Http.Action: ApiControllerActionSelector SelectAction 
02-12 19:27:37 Info ApiControllerActionSelector SelectAction Selected action 'Configure()' 
System.Web.Http.Action: ApiControllerActionSelector SelectAction Selected action 'Configure()'
02-12 19:27:37 Info JackettAuthorizedAttribute OnAuthorizationAsync  
System.Web.Http.Filters: JackettAuthorizedAttribute OnAuthorizationAsync 
02-12 19:27:37 Info JackettAuthorizedAttribute OnAuthorizationAsync  
System.Web.Http.Filters: JackettAuthorizedAttribute OnAuthorizationAsync 
02-12 19:27:37 Info HttpActionBinding ExecuteBindingAsync  
System.Web.Http.ModelBinding: HttpActionBinding ExecuteBindingAsync 
02-12 19:27:37 Info HttpActionBinding ExecuteBindingAsync  
System.Web.Http.ModelBinding: HttpActionBinding ExecuteBindingAsync 
02-12 19:27:37 Info JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Configure()' 
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Configure()'
02-12 19:27:37 Info JackettAPINoCacheAttribute OnActionExecutingAsync  
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutingAsync 
02-12 19:27:37 Info ApiControllerActionInvoker InvokeActionAsync Action='Configure()' 
System.Web.Http.Action: ApiControllerActionInvoker InvokeActionAsync Action='Configure()'
02-12 19:27:37 Info ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Configure()' 
System.Web.Http.Action: ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Configure()'
02-12 19:27:37 Debug IWebClient.GetString(Url:https://www.torrentbytes.net/) 
02-12 19:27:38 Debug IWebClient: Returning Redirect => https://www.torrentbytes.net/login.php?returnto=%2F  
02-12 19:27:38 Debug updating Cookies  => PHPSESSID=fh8sgqsargvh3mpg264h5evpm6; checksum=deleted; uid=deleted; pass=deleted; validation=deleted 
02-12 19:27:38 Debug IWebClient.GetString(Url:https://www.torrentbytes.net/takelogin.php) 
02-12 19:27:38 Debug UnixLibCurlWebClient: Posting username=username&password=-removed-%40&returnto=%2F&login=Log+in%21 
02-12 19:27:39 Debug IWebClient: Returning OK => <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Torrentbytes :: </title>
<link type="text/css" rel="stylesheet" href="themes/classic/css/A.theme.css.pagespeed.cf.BNqzQ18isj.css"/>
<link rel="shortcut icon" type="image/png" href="themes/classic/static/favicon.png.pagespeed.ce.2-hjEAtDzn.png"/>
</head>
<body><noscript><meta HTTP-EQUIV="refresh" content="0;url='https://www.torrentbytes.net/takelogin.php?PageSpeed=noscript'" /><style><!--table,div,span,font,p{display:none} --></style><div style="display:block">Please click <a href="https://www.torrentbytes.net/takelogin.php?PageSpeed=noscript">here</a> if you are not redirected within a few seconds.</div></noscript>
<div id="bannerWrapper">
<div id="logo"></div>
</div>
<div id="innerwrapper">
<div id="borderTop"></div>
<div id="leftBorder"><img class="sideBorderImg" alt="" src="themes/classic/static/sidebar_left.png.pagespeed.ce.nQh0Ar3JUI.png"/></div>
<div id="rightBorder"><img class="sideBorderImg" alt="" src="themes/classic/static/sidebar_right.png.pagespeed.ce.VS4pubDXiB.png"/></div>
<div id="top">
<div id="menu">
<img alt="" src="themes/classic/static/menu_seperator.png.pagespeed.ce.eiOA-YatXt.png" class="seperator"/>
<table class="mainMenu">
<tr>
<td align="center" class="navigation"><a href="signup.php">Signup</a></td>
<td align="center" class="navigation"><a href="login.php">Login</a></td>
<td align="center" class="navigation"><a href="rules.php">Rules</a></td>
<td align="center" class="navigation"><a href="faq.php">FAQ</a></td>
</tr>
</table>
<table class="mainMenu">
<tr>
</tr>
</table>
<img alt="" src="themes/classic/static/menu_seperator.png.pagespeed.ce.eiOA-YatXt.png" class="seperator"/>
</div>
</div>
<div id="mainframe">
<div id="content">
<div align="center">
<table class=main width=750 border=0 cellpadding=0 cellspacing=0><tr><td class=embedded>
<h2>Login failed!</h2>
<table width=100% border=1 cellspacing=0 cellpadding=10><tr><td class=text>
Username or password incorrect</td></tr></table></td></tr></table>
<br/>
</div>
</div>
</div>
</div>
<div id="bottom"></div>
<div class="bottomWrapper">
<div id="bottomLeft"></div>
<div id="bottomRight"></div>
</div>
</body>
</html>
 
02-12 19:27:39 Info ReflectedHttpActionDescriptor ExecuteAsync Action returned 'System.Web.Http.Results.JsonResult`1[Newtonsoft.Json.Linq.JObject]' 
System.Web.Http.Action: ReflectedHttpActionDescriptor ExecuteAsync Action returned 'System.Web.Http.Results.JsonResult`1[Newtonsoft.Json.Linq.JObject]'
02-12 19:27:39 Info ApiControllerActionInvoker InvokeActionAsync  
System.Web.Http.Action: ApiControllerActionInvoker InvokeActionAsync 
02-12 19:27:39 Info JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Configure()' 
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Configure()'
02-12 19:27:39 Info JackettAPINoCacheAttribute OnActionExecutedAsync  
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutedAsync 
02-12 19:27:39 Info AdminController ExecuteAsync  
System.Web.Http.Controllers: AdminController ExecuteAsync 
{"name":"TorrentBytes","result":"error","error":"","config":[{"id":"sitelink","type":"inputstring","name":"Site Link","value":"https://www.torrentbytes.net/"},{"id":"username","type":"inputstring","name":"Username","value":"username"},{"id":"password","type":"inputstring","name":"Password","value":"password"},{"id":"","type":"displayinfo","name":"","value":null},{"id":"cookieheader","type":"hiddendata","name":"CookieHeader","value":""},{"id":"lasterror","type":"hiddendata","name":"LastError","value":""}]}
02-12 19:27:39 Debug Response: {"name":"TorrentBytes","result":"error","error":"","config":[{"id":"sitelink","type":"inputstring","name":"Site Link","value":"https://www.torrentbytes.net/"},{"id":"username","type":"inputstring","name":"Username","value":"username"},{"id":"password","type":"inputstring","name":"Password","value":"password"},{"id":"","type":"displayinfo","name":"","value":null},{"id":"cookieheader","type":"hiddendata","name":"CookieHeader","value":""},{"id":"lasterror","type":"hiddendata","name":"LastError","value":""}]} 
02-12 19:27:39 Info WebAPIRequestLogger SendAsync  
System.Web.Http.MessageHandlers: WebAPIRequestLogger SendAsync 
02-12 19:27:39 Info   Content-type='application/json; charset=utf-8', content-length=514 
System.Web.Http.Request:   Content-type='application/json; charset=utf-8', content-length=514
02-12 19:27:39 Info AdminController Dispose  
System.Web.Http.Controllers: AdminController Dispose 
02-12 19:27:39 Info AdminController Dispose  
System.Web.Http.Controllers: AdminController Dispose 
```

**Comment by kaso17** [comment]
It's working fine for me. 
It's failing with "Username or password incorrect". please double check them.
And please also change your password as you posted it as part of the log (I removed it by now).
Not sure if the special character in your password is causing the problem (if it does please let us know).

**Comment by adamwinn** [comment]
I got it working now.  It'd be nice if it said "Configuration failed:  Username or password incorrect" instead of just "Configuration failed: "

**Comment by kaso17** [comment]
Fixed with 163d21629d662535fcafac4cd0c3bd21d1fb2b09
